Folly/folly/portability/Builtins.h

```diff
@@ -74,7 +74,7 @@ FOLLY_ALWAYS_INLINE int __builtin_ctzl(unsigned long x) {
   return __builtin_ctz((unsigned int)x);
 }
 
-#if defined(_M_IX86) || defined(_M_ARM)
+#if defined(_M_IX86) || defined(_M_ARM) || defined(_M_ARM64)
```

Comment:
>#if defined [](start = 0, length = 11)

Could you add a quick comment of why these differences are needed between platforms?